applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -325,6 +317,20 @@ static void button_evt_handler(struct button_evt event)
 
 		break;
 
+	case BUTTON_5:
+#if !(CONFIG_AUDIO_MUTE)
+		ret = le_audio_user_defined_button_press(LE_AUDIO_USER_DEFINED_ACTION_2);
+		if (ret) {
+			LOG_WRN("User defined button 5 press failed");
+		}
+#else
+		ret = le_audio_volume_mute();
+		if (ret) {
+			LOG_WRN("Failed to mute volume");
+		}
+#endif
```

Comment:
Agreed, changed

Reply:
Why has it change to ..._ACTION_2 ?
I believe it would be clearer if this was ..._ACTION_5 as it is button 5. I know we currently do not have additional actions on buttons 0-4 but this will change once we have long presses, quick multiple presses, etc. Hence we would have actions aligned to the button and yes I know the Bluetooth module knows nothing about the buttons but I feel it aids the reading/understanding of the code.

Reply:
We should aim not to create code that is tied that hard to our audio-DK. A customer using this code might not have the same button layout, so since we have two user-defined actions, it makes sense to call this user_defined_action_2.